media/css/pebbles/global.scss

```diff
@@ -8,6 +8,7 @@
 
 @import
     // Fonts
+    'includes/fonts/zilla-slab',
```

Comment:
As we're adding this `@import` here, the existing imports can now also be removed from the existing hub pages?

- `media/css/firefox/features/common.scss`
- `media/css/firefox/hub/home.scss`
- `media/css/firefox/product-page.scss`